tests/ambsym/tresolution_identifier_local.nim

```diff
@@ -0,0 +1,32 @@
+discard """
+labels: "identifier local overload proc resolution"
+description: '''
+. Test overloading of procs with locals
+'''
+"""
+
+type
+  TMyType = object
+    len: int
+    data: string
+
+proc len(x: TMyType): int {.inline.} = return x.len
+proc data(x: TMyType): string {.inline.} = return x.data
+
+proc doLen(s: TMyType, len: int) : int =
+  return len( s )
+proc doData(s: TMyType, len: int) : string =
+  return data( s )
+
+
+var
+  m: TMyType
+
+const size = 7
+const content = "1234"
+
+m.len = size
+m.data = content
+
+doAssert doLen(m, 5) == size
```

Comment:
I'm not too keen on what was being tested here to begin with. 
Is it the overloading of the `len` stdlib proc to our own type? or is it the disambiguation of the x.len (field) vs x.len (UFCS)?
Why does `doLen` even has a second argument?
Does the {.inline.} pragma change the semantics?